samples/RhythmGame/src/main/cpp/audio/Player.cpp

```diff
@@ -19,30 +19,41 @@
 
 void Player::renderAudio(int16_t *targetData, int32_t numFrames){
 
+    const int32_t channelCount = mSource.getChannelCount();
```

Comment:
Do we guarantee somehow that the AAudio stream channel count matches the channel count of the source files?

Reply:
We don't. However, this is documented here: https://github.com/google/oboe/tree/master/samples/RhythmGame#caveats-and-limitations